ovs/lib/vdisk.py

```diff
@@ -780,6 +791,8 @@ def set_config_params(vdisk_guid, new_config_params):
                         block_size = vol_info.lba_size * vol_info.cluster_multiplier or 4096
                         limit = new_value * 1024 * 1024 * 1024 / block_size if new_value else None
                         vdisk.storagedriver_client.set_readcache_limit(volume_id, limit)
+                    elif key =='metadata_cache_size':
+                        vdisk.storagedriver_client.set_metadata_cache_capacity(str(vdisk.volume_id), int(new_value / METADATA_CACHE_PAGE_SIZE))
```

Comment:
str(vdisk.volume_id) can be replaced by volume_id (similar to the other set methods)
int(new_value / METADATA...) is obsolete cast, new_value has been checked by verify_required_params to be an int and METADATA... is an int